src/main/java/seedu/todo/commons/core/Config.java

```diff
@@ -37,28 +38,12 @@ public void setLogLevel(Level logLevel) {
         this.logLevel = logLevel;
     }
 
-    public String getUserPrefsFilePath() {
-        return userPrefsFilePath;
+    public String getDatabaseFilePath() {
+        return databaseFilePath;
     }
 
-    public void setUserPrefsFilePath(String userPrefsFilePath) {
-        this.userPrefsFilePath = userPrefsFilePath;
-    }
-
-    public String getAddressBookFilePath() {
-        return addressBookFilePath;
-    }
-
-    public void setAddressBookFilePath(String addressBookFilePath) {
-        this.addressBookFilePath = addressBookFilePath;
-    }
-
-    public String getAddressBookName() {
-        return addressBookName;
-    }
-
-    public void setAddressBookName(String addressBookName) {
-        this.addressBookName = addressBookName;
+    public void setDatabaseFilePath(String databaseFilePath) {
+        this.databaseFilePath = databaseFilePath;
```

Comment:
Should we try to detect if there is a ".json" in the specified path and if not, add it in? Reason for this is that if the user enters something like "hello", the storage file will be stored without a file extension. But we want the file extension there because it is after all a json. If the user manages to store the file as some dependency like .java or .class or idk, we are in trouble.